packages/destination-actions/src/destinations/insider/insider-helpers.ts

```diff
@@ -253,3 +253,165 @@ export function sendTrackEvent(
 
   return { users: [payload] }
 }
+
+export function bulkUserProfilePayload(data: UserPayload[]) {
+  const batchPayload = data.map((userPayload) => {
+    const identifiers = {
+      uuid: userPayload.uuid,
+      custom: {
+        segment_anonymous_id: userPayload.segment_anonymous_id
+      }
+    }
+
+    if (userPayload.email_as_identifier) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
It's not a good idea to mute eslinter, but as this bug as urgent, you can proceed

Reply:
yes it is, but most mutes about types, we can not add types manually, because of that it giving this errors.